### PR TITLE
add give_reports_graph_additional_stats to individual form stats

### PR DESCRIPTION
**Pull request by macbookandrew**
## Description
Add `give_reports_graph_additional_stats` action to specific form reports.

## How Has This Been Tested?
I added an action hook in my theme’s functions.php to insert another row into the table, and it worked exactly as expected.

## Types of changes
- New feature: in that it’s a non-breaking change which adds functionality
- Bugfix: in that it adds feature parity between default reports view and specific form report view

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code has proper inline documentation.

### CODE REVIEW

File: includes/admin/reports/graphing.php [hunk 1]
```diff
@@ -469,6 +469,18 @@ function give_reports_graph_of_form( $form_id = 0 ) {
...
 				</tbody>
 			</table>
+
+			<?php
+			/**
+			 * Fires on report graphs widget.
+			 *
+			 * Allows you to add additional stats to the widget.
+			 *
+			 * @since 1.0
```
Comment (DevinWalker) [inline includes/admin/reports/graphing.php]: @macbookandrew this version is incorrect

**Comment by macbookandrew** [comment]
@DevinWalker Is 2.2.1 what you want? (I figured since the action was introduced in 1.0 in the other widget, 1.0 might still be appropriate.)

**Comment by DevinWalker** [comment]
No worries, this is fine. I'll update it.

**Comment by macbookandrew** [comment]
@DevinWalker Thanks. Just curious: what version do you anticipate this will be released in?

**Pull request merged**